### ISSUE TITLE
Как перестроить блок по-другому

### ISSUE THREAD

**Issue by itgirlschool** [Lartwoods/Week7-to-check#6]
Когда в коде видны значения -50%, -100% - т.д., то как правило, это костыль, который можно поправить как-то проще с помощью css - посмотри, почему тебе нужно делать такие большие отступы и попробуй решить проблему по-другому)
https://github.com/Lartwoods/Week7-to-check/blob/2270743648c1dd5a9734778202961b461874bf6d/week7_task2/plantshopstyle_ad.css#L247

**Comment by Lartwoods** [comment]
Да, я понимаю, не успевала совсем, и просто уже поставила марджины, чтоб хоть не так сильно разъезжалось. Я до сих пор не совсем понимаю, почему контент настолько разъезжается сильно друг от друга? Улетает вниз просто, хотя, вроде, величины относительные. Или от чего это зависит конкретно в данном коде? Не использовала mobile first пока, наверное с ним меньше таких косяков в том числе.
